packages/fluentui/react-northstar/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -63,23 +63,23 @@ export const radioGroupItemStyles: ComponentSlotStylesPrepared<RadioGroupItemSty
 
   indicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
     margin: `0 ${pxToRem(12)} 0 0`,
-    borderRadius: '50%',
-    borderWidth: '1px',
-    borderStyle: 'solid',
-    borderColor: v.indicatorBorderColorDefault,
+    outline: 0,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
     width: pxToRem(12),
     height: pxToRem(12),
+    verticalAlign: 'midddle',
+    color: v.indicatorBorderColorDefault,
```

Comment:
Can we rename these variables to not contain the `border` in their name?

Reply:
Done